.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "ext/tezos"]
 	path = ext/tezos
 	url = https://gitlab.com/tezos/tezos.git
+[submodule "ext/pandoc-tangle"]
+	path = ext/pandoc-tangle
+	url = https://github.com/ehildenb/pandoc-tangle.git
+[submodule "ext/blockchain-k-plugin"]
+	path = ext/blockchain-k-plugin
+	url = https://github.com/sskeirik/blockchain-k-plugin
```

Comment:
You guys should _not_ depend on this fork. You should use the https://github.com/runtimeverification/blockchain-k-plugin version. If you need changes to the blockchain-k-plugin, you need to get them reviewed, tested, and merged into `master` before you can update this submodule to use those changes.

Reply:
We have lost a lot of time in the past to people not following this rule, so it's worth it to just get it right the first time.